### PR TITLE
[WIP] [Feature] unified toolbar for Files and Project Organizer

### PR DESCRIPTION
**Pull request by caneruguz**
## Purpose
Related to the issue in discussion #2282 and several other issues and requests. The goal is to create a unified look and logic for the way row specific actions happen in Treebeard variations. Specific Goals
- Files page: Carry over action column to a toolbar
- Project Organizer: Unite toolbar look and layout with Files
- Add multiselect to Files page
- Delete Multiple files within files page - #1220 
- Cancel All Uploads or individual uploads - #1688 
- Unify the color scheme to be consistent - #1693 

## Changes
- Files page has action column removed, instead has a toolbar with contextual buttons showing up
- Project organizer code is refactored to turn its toolbar from a mako/js to Treebeard toolbar
- Delete multiselect feature is added 
- Cancel uploads feature is added. 
...

## Side Effects
...

### CODE REVIEW

**Comment by caneruguz** [comment]
Items to be added after review from @brianjgeiger : 
- When you click one row, based on what that row is, other rows that can’t be clicked should be greyed out so user does not even start the attempt. 
- If there are check boxes, they should work like check boxes, i.e clicking one turns it on, clicking others does not turn it off again.

File: website/static/js/fangorn.js [hunk 1]
```diff
@@ -49,6 +49,64 @@ var getExtensionIconClass = function(name) {
...
+        }
+    }
+    return item;
+}
+
+function cancelUploads (row) {
+    var treebeard = this;
+    var filesArr = treebeard.dropzone.getQueuedFiles();
+    for (var i = 0; i < filesArr.length; i++) {
+        var m = filesArr[i];
```
Comment (sloria) [inline website/static/js/fangorn.js]: This is shadowing the outer `m` variable (mithril). Rename this.

File: website/static/js/fangorn.js [hunk 2]
```diff
@@ -49,6 +49,64 @@ var getExtensionIconClass = function(name) {
...
+        }
+        if (child.data.tmpID === tmpID) {
+            item = child;
+        }
+    }
+    return item;
+}
+
+function cancelUploads (row) {
+    var treebeard = this;
```
Comment (sloria) [inline website/static/js/fangorn.js]: > Use `self` to save a reference to this.

https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html

Comment (caneruguz) [inline website/static/js/fangorn.js]: Can we make an exception to this rule in this case? I'm using treebeard to make it clear to developers that this refers to the Treebeard API.

Comment (sloria) [inline website/static/js/fangorn.js]: Sure, I think that makes sense.

**Comment by caneruguz** [comment]
Closing this, will reopen with unified add folder branch that has folder ui in it as well.

**Pull request closed without merging**